tests/system/README.md

```diff
@@ -53,7 +53,14 @@ For example, to run only with the txtar files in/under `testdata/*foo*/*bar*`,
 including `testdata/*foo*/*bar*.txtar`:
 
 ```
-gotestsum -f testname -run /foo/bar
+gotestsum -f testname ./tests/system -run /foo/bar
```

Comment:
I think you should give a concrete example here, not just `/foo/bar`

Reply:
I agree, no `foo`, `bar` and friends in any user facing documentation.

Reply:
done